watertap/property_models/NaCl_T_dep_prop_pack.py

```diff
@@ -1255,9 +1190,9 @@ def rule_solubility_comp(b, j):
 
     def _pressure_sat(self):
         self.pressure_sat = Var(
-            initialize=0.01,
-            bounds=(0.0, 2),
-            units=pyunits.MPa,
+            initialize=1e3,
+            bounds=(1, 1e8),
```

Comment:
Are these bounds attributed to the valid range of the fitted relationship? Or are these fairly arbitrary?

Reply:
These are fairly arbitrary. I have been using the seawater prop pack as a guide for how tight or loose to set bounds because I was having some difficulty when attempting to set up an example using a unit model. With the bounds more closely aligned to the seawater prop pack bounds, I was able to get a running example with RO.